lib/accessly/policy/base.rb

```diff
@@ -54,27 +54,36 @@ def segment_id
         nil
       end
 
-      def grant(action, object = nil)
-        object_id = _get_object_id(object)
+      def can?(action, object = nil)
+        action_method_name = _resolve_action_method_name("#{action}?")
+        if (action_method_name.nil?)
+          object_id = _get_object_id(object)
+          action_id = _get_action_id(action, object_id)
+          _can_do_action?(action, action_id, object)
+        else
+          send(action_method_name, object)
+        end
+      end
 
-        action_id = if object_id.nil?
-          _get_general_action_id!(action)
+      def list(action)
```

Comment:
Same question in comment above

Reply:
Likewise, I believe you can just do this:

```rb
def list(action)
  send(action)
end
```